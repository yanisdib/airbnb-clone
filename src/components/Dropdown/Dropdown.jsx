import styled from 'styled-components';


export default styled.div`
    position: absolute;
    width: ${({ width }) => width ? `${width}%` : '100%'};
    min-height: 200px;
    top: 60px;
    right: 0;
    background-color: #FFFFFF;
    border-radius: 12px; 
    z-index: 999;
`;