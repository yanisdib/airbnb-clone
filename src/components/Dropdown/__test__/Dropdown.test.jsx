import { shallow } from 'enzyme';

import Dropdown from '../Dropdown';


describe('Dropdown component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Dropdown />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})