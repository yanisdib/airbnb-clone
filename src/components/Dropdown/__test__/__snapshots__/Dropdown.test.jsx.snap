// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown component renders correctly without crashing 1`] = `ShallowWrapper {}`;
